suites/mpm/taichi/mpm_compile.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+# -- stdlib --
+from typing import Dict
+import tempfile
+
+# -- third party --
+import taichi as ti
+from .mpm import MPM
+
+# -- code --
+class MPMCompile(MPM):
+    name = 'mpm_compile'
+    
+    @property
+    def repeats(self):
+        return 2
+
+    def init(self, n_grid, dim):
+        super().init(n_grid, dim)
+        
+        arch = ti.lang.impl.get_runtime().prog.config().arch
+        self.mod = ti.aot.Module(arch) 
+
+    def run_iter(self):
+        for _ in range(self.steps):
```

Comment:
Are you intending to add the `self.mpm_substep` kernel `self.step` times here? The code here looks like a casual modification of the original MPM code.